frontend/.eleventy.js

```diff
@@ -2,13 +2,18 @@
  * sri sri guru gaurangau jayatah
  */
 
+require('dotenv').config({
+  path: `${__dirname}/.env.${process.env.NODE_ENV}.local`,
+});
```

Comment:
1. Use `development` as a fallback value, because as you noticed, `NODE_ENV` can be undefined.
2. Load also `.env.local` before this, and normal `.env` even before this.